spot_rl_experiments/spot_rl/real_policy.py

```diff
@@ -120,7 +200,43 @@ def __init__(self, checkpoint_path, device):
             }
         )
         action_space = spaces.Box(-1.0, 1.0, (4,))
-        super().__init__(checkpoint_path, observation_space, action_space, device)
+        super().__init__(
+            checkpoint_path,
+            observation_space,
+            action_space,
+            device,
+        )
+
+
+class MobileGazePolicy(RealPolicy):
+    def __init__(self, checkpoint_path, device):
+        observation_space = SpaceDict(
+            {
+                "arm_depth_bbox_sensor": spaces.Box(
+                    low=np.finfo(np.float32).min,
+                    high=np.finfo(np.float32).max,
+                    shape=(240, 228, 1),
+                    dtype=np.float32,
+                ),
+                "articulated_agent_arm_depth": spaces.Box(
+                    low=0.0, high=1.0, shape=(240, 228, 1), dtype=np.float32
+                ),
+                "joint": spaces.Box(
+                    low=np.finfo(np.float32).min,
+                    high=np.finfo(np.float32).max,
+                    shape=(4,),
+                    dtype=np.float32,
+                ),
+            }
+        )
```

Comment:
@jimmytyyang 
Can we test & check if we can use the same observation_space dict keys as that of static gaze ? Just so we don't want to remap then at run-time?

Reply:
Hi @KavitShah1998 @tusharsangam! After investigation, the name for the observation keys needs to be the same as the one when we do conversion from Hab3 to torchscript. Since when do conversion, we need to use the exact name when we train the policy to do conversion, it is not possible to change the name of observation keys. And I think we should use the consistent name for observations nevertheless since we make sure there is no confusion